### PR TITLE
Use viewReuseId to improve feed performance in sample app #63

### PR DESCRIPTION
**Pull request by nicksnyder**

### CODE REVIEW

File: LayoutKitSampleApp/DWURecyclingAlert.m [hunk 1]
```diff
@@ -0,0 +1,468 @@
+//DWURecyclingAlert.m
+//Copyright (c) 2015 Di Wu
+//
+//Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (plivesey) [inline LayoutKitSampleApp/DWURecyclingAlert.m]: I don't see this actually used anywhere?

Comment (nicksnyder) [inline LayoutKitSampleApp/DWURecyclingAlert.m]: It does swizzling auto magic. All you have to do is include it in the project.

Comment (plivesey) [inline LayoutKitSampleApp/DWURecyclingAlert.m]: Maybe include in the commit message why this is needed?

**Comment by plivesey** [comment]
Oh. Nvm, you merged. (couldn't work out how to approve, lol)

**Pull request merged**